core/store/presenters/presenters.go

```diff
@@ -115,6 +117,8 @@ func NewConfigPrinter(store *store.Store) (ConfigPrinter, error) {
 	if config.ExplorerURL() != nil {
 		explorerURL = config.ExplorerURL().String()
 	}
+	p2pBootstrapPeers, _ := config.P2PBootstrapPeers(nil)
```

Comment:
`if errors.Cause(err) == ErrUnset { do nothing } else { log }`
Future proofing

Reply:
hmm the envprinter right now uses reflection to loop over all the fields in `EnvPrinter` to print them. So we'd have to add a special case for this to do what you suggest. Not sure its worth it vs just showing an empty slice on ErrUnset?